core/kubernetes/k8s.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ "$1" == "up" ]; then
+    kubectl apply -f ./openhim
+    kubectl apply -f ./hapi-fhir
+    kubectl get services
+    kubectl get ingress
+
+    # create HOST entry for ingress
+    sudo sed -i "/HOST alias for kubernetes Minikube/d" /etc/hosts
+    echo -e "\n$(minikube ip) $(kubectl get ingress -o jsonpath="{..host}") # HOST alias for kubernetes Minikube" | sudo tee -a /etc/hosts
+elif [ "$1" == "down" ]; then
+    kubectl delete deployment openhim-console-deployment
+    kubectl delete deployment openhim-core-deployment
+    kubectl delete deployment hapi-fhir-server-deployment
+    kubectl delete deployment hapi-fhir-mysql-deployment
+elif [ "$1" == "destroy" ]; then
+    kubectl delete deployment --all --grace-period 2
+    kubectl delete service --all --grace-period 2
+    kubectl delete configmap --all --grace-period 2
+    kubectl delete persistentvolumeclaims --all --grace-period 2
+    kubectl delete cronjobs --all --grace-period 2
+    kubectl delete pods --all --grace-period 2
+    kubectl delete ingress --all --grace-period 2
```

Comment:
You can delete all these in one go by pointing to the kustomization file folder. E.g. `kubectl delete -k .`